Cloudinary/Shared/Coudinary.NetCoreShared/CloudinaryShared.cs

```diff
@@ -6,9 +6,9 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Coudinary.NetCoreShared
+namespace CloudinaryShared.Core
 {
-    public class CloudinaryShared
+    public class CloudinaryShared<TApi> where TApi: ApiShared
```

Comment:
Yes, sorry, we'll include it in our next commit. There is a need to only add 'abstract' before class definition. No more additional changes needed - it's been done earlier.